src/UI/Areas/Identity/Pages/Account/ConfirmEmail.cshtml.cs

```diff
@@ -3,11 +3,13 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace Microsoft.AspNetCore.Identity.UI.Pages.Account.Internal
 {
+    [AllowAnonymous]
```

Comment:
While testing realized that this required the AllowAnonymous attribute for global authorization scenario as well